change/@ni-nimble-components-5067d35f-244c-42ff-b498-4329212f03ce.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
Since this component is still marked experimental and no client is using it, we can keep the change type as `patch` so existing Nimble clients realize that this change doesn't include APIs that affect them.